public/app/features/dashboard/components/SubMenu/SubMenuCtrl.ts

```diff
@@ -1,16 +1,22 @@
 import angular, { ILocationService } from 'angular';
 import _ from 'lodash';
 import { VariableSrv } from 'app/features/templating/all';
+import { CoreEvents } from '../../../../types';
 
 export class SubMenuCtrl {
   annotations: any;
   variables: any;
   dashboard: any;
+  submenuEnabled: boolean;
 
   /** @ngInject */
   constructor(private variableSrv: VariableSrv, private $location: ILocationService) {
     this.annotations = this.dashboard.templating.list;
     this.variables = this.variableSrv.variables;
+    this.submenuEnabled = false;
+    this.dashboard.events.on(CoreEvents.submenuVisibilityChanged, (enabled: boolean) => {
```

Comment:
I was thinking this lived in the React SubMenu component, and we did not render any angular component in the case of submenu disabled. But think this is also fine, hopefully we will rewrite this to react soon.

Reply:
Ok, I didn't catch that earlier.